meshnets/utils/datasets.py

```diff
@@ -42,7 +42,7 @@ class FromDiskGeometricDataset(Dataset):
     def __init__(self,
                  data_dir: str,
                  *args,
-                 mesh_file_name: str = 'pressure_field.vtk',
+                 mesh_file_name: str = 'pressure_field_interpolated.vtk',
```

Comment:
I guess the graph we will call it the same?

Reply:
Yes, since we only have one graph for training. But I see your point. We should maybe also rename the graphs in the pre-processing step